.gitignore

```diff
@@ -1,9 +1,8 @@
-.vscode
 .venv
 __pycache__
 .pytest_cache
 dist
 
 # Miscellaneous
 test.py
-*.afphoto
\ No newline at end of file
+*.afphoto
```

Comment:
Is this necessary?

Reply:
Ah good spot - It was from previously establishing icons - I guess similarly with `test.py` was initially for local testing and could just be handled with better developer workflow

Reply:
Could replace the whole gitignore with one generated by gitignore.io?
https://www.toptal.com/developers/gitignore/api/python

Reply:
The guiding principle I personally use is to keep the gitignore restricted to any files we expect to be generated by Python or the tooling that is integrated into the repo (so, if we decide that we're using black we would want to ignore any files that black creates). 

.idea and .vscode don't come under that because they're not repo-specific tooling - developers should be encouraged to add that to their personal, global gitignore. 

All this is to say I agree with your second suggestion.